### PR TITLE
MET-2018_MET-5083 simplify case check invalid url dereference

### PR DESCRIPTION
**Pull request by jeortizquan**

### CODE REVIEW

File: metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImpl.java [hunk 1]
```diff
@@ -78,115 +70,6 @@ private static URL checkIfUrlIsValid(HashSet<Report> reports, String id) {
     }
   }
 
-  private static HttpURLConnection getClient(URL checkedUrl) throws IOException {
```
Comment (jochen-vermeulen) [inline metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImpl.java:81]: There is a constant `public static final String HTTPS = "https";` that is no longer used.

File: metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImplTest.java [hunk 2]
```diff
@@ -60,9 +57,9 @@
 public class DereferencerImplTest {
 
   private static final String[] DEREFERENCE_EXTRACT_RESULT_INVALID = {
-      "http://invalid-example.host/about",
-      "http://invalid-example.host/concept",
-      "http://invalid-example.host/place"};
```
Comment (jochen-vermeulen) [inline metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImplTest.java:65]: Perhaps a good check would be a space character in the URL (either in the path or in the query): this happens a lot and should trigger a warning.

File: metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImpl.java (hunk 1 above)
Comment (jeortizquan) [reply metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImpl.java:81]: removed.

File: metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImplTest.java (hunk 2 above)
Comment (jeortizquan) [reply metis-enrichment/metis-enrichment-client/src/test/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImplTest.java]: done. added URI checking as well.

**Pull request merged**